### PR TITLE
[hi] Add content/hi/docs/setup/production-environment/windows/user-guide-windows-containers.md

### PR DESCRIPTION
**Pull request by arsheelsheikh**
Localizes

- `content/en/docs/setup/production-environment/windows/user-guide-windows-containers.md`

### CODE REVIEW

**Comment by arsheelsheikh** [comment]
This PR is in continuation of hi localization effort: #29353

/language hi

**Comment by arsheelsheikh** [comment]
@anubha-v-ardhan

**Comment by anubha-v-ardhan** [comment]
/retitle [hi] Add content/hi/docs/setup/production-environment/windows/user-guide-windows-containers.md

File: content/hi/docs/setup/production-environment/windows/user-guide-windows-containers.md [hunk 1]
```diff
@@ -0,0 +1,297 @@
+---
+reviewers:
+- jayunit100
+- jsturtevant
+- marosset
+- perithompson
```
Comment (anubha-v-ardhan) [inline content/hi/docs/setup/production-environment/windows/user-guide-windows-containers.md:6]:
```suggestion
```

File: content/hi/docs/setup/production-environment/windows/user-guide-windows-containers.md [hunk 2]
```diff
@@ -0,0 +1,297 @@
...
+
+<!-- overview -->
+
+Windows एप्लिकेशन कई संगठनों में चलने वाली सेवाओं और अनुप्रयोगों के एक बड़े हिस्से का गठन करते हैं।
+यह मार्गदर्शिका आपको कुबेरनेट्स में Windows कंटेनर को कॉन्फ़िगर और परिनियोजित करने के चरणों के बारे में बताती है।
+
+
+<!-- body -->
+
+## उद्देश्यों
```
Comment (anubha-v-ardhan) [inline content/hi/docs/setup/production-environment/windows/user-guide-windows-containers.md:20]:
```suggestion
## उद्देश्य
```

**Review by anubha-v-ardhan — CHANGES_REQUESTED** [review]
Initial review

**Comment by anubha-v-ardhan** [comment]
Can you please sign the CLA so that your PR can be reviewed : )
https://github.com/kubernetes/community/blob/master/CLA.md

**Review by vedant-kakde — CHANGES_REQUESTED** [review]

**Comment by kaiwalyakoparkar** [comment]
@arsheelsheikh Could you please convert this PR into draft PR please?

**Comment by arsheelsheikh** [comment]
> @arsheelsheikh Could you please convert this PR into draft PR please?

Thanks, Done.

**Review by Arhell — COMMENTED** [review]
/check-cla

**Comment by arsheelsheikh** [comment]
"I signed it"

File: content/hi/docs/setup/production-environment/windows/user-guide-windows-containers.md [hunk 3]
```diff
@@ -0,0 +1,292 @@
...
+<!-- body -->
+
+## उद्देश्य
+
+* Windows नोड पर Windows कंटेनर चलाने के लिए एक उदाहरण परिनियोजन कॉन्फ़िगर करें
+* (वैकल्पिक) ग्रुप मैनेज्ड सर्विस अकाउंट्स (जीएमएसए) का उपयोग करके अपने पॉड के लिए एक एक्टिव डायरेक्टरी पहचाने और उसे कॉन्फ़िगर करें
+
+## शुरू करने से पहले
+
+* एक कुबेरनेट्स क्लस्टर बनाएं जिसमें शामिल है a
```
Comment (vedant-kakde) [inline content/hi/docs/setup/production-environment/windows/user-guide-windows-containers.md:22]:
```suggestion
* एक कुबेरनेट्स क्लस्टर बनाएं जिसमें
```

File: content/hi/docs/setup/production-environment/windows/user-guide-windows-containers.md [hunk 4]
```diff
@@ -0,0 +1,292 @@
...
+नियंत्रण विमान और [worker node running Windows Server](/docs/tasks/administer-cluster/kubeadm/adding-windows-nodes/)
```
Comment (vedant-kakde) [inline content/hi/docs/setup/production-environment/windows/user-guide-windows-containers.md:23]:
```suggestion
नियंत्रण विमान और [worker node running Windows Server](/docs/tasks/administer-cluster/kubeadm/adding-windows-nodes/) शामिल है |
```

File: content/hi/docs/setup/production-environment/windows/user-guide-windows-containers.md [hunk 5]
```diff
@@ -0,0 +1,292 @@
...
+* यह ध्यान रखना महत्वपूर्ण है कि कुबेरनेट्स पर सेवाओं और वर्कलोड को बनाना और तैनात करना
+Linux और विंडोज कंटेनरों के लिए समान रूप से व्यवहार करता है। 
+[Kubectl commands](/docs/reference/kubectl/overview/) क्लस्टर के साथ इंटरफेस करने के लिए समान हैं।
```
Comment (vedant-kakde) [inline content/hi/docs/setup/production-environment/windows/user-guide-windows-containers.md:26]:
```suggestion
[kubectl commands](/docs/reference/kubectl/overview/) क्लस्टर के साथ इंटरफेस करने के लिए समान हैं।
```

File: content/hi/docs/setup/production-environment/windows/user-guide-windows-containers.md [hunk 6]
```diff
@@ -0,0 +1,292 @@
...
+विंडोज कंटेनर के साथ अपने अनुभव को जम्पस्टार्ट करने के लिए नीचे दिए गए अनुभाग में उदाहरण दिया गया है।
```
Comment (vedant-kakde) [inline content/hi/docs/setup/production-environment/windows/user-guide-windows-containers.md:27]:
```suggestion
Windows कंटेनर के साथ अपने अनुभव को जम्पस्टार्ट करने के लिए नीचे दिए गए अनुभाग में उदाहरण दिया गया है।
```

File: content/hi/docs/setup/production-environment/windows/user-guide-windows-containers.md [hunk 7]
```diff
@@ -0,0 +1,292 @@
...
+    ```bash
+    kubectl apply -f win-webserver.yaml
+    kubectl get pods -o wide -w
+    ```
+
+    जब सेवा को सही ढंग से तैनात किया जाता है, तो दोनों पॉड्स को रेडी के रूप में चिह्नित किया जाता है। वॉच कमांड से बाहर निकलने के लिए, Ctrl + C दबाएं।
+
+1. जांचें कि परिनियोजन सफल हुआ। सत्यापित करना:
+
+    * विंडोज नोड पर प्रति पॉड दो कंटेनर, उपयोग करें `docker ps`
```
Comment (vedant-kakde) [inline content/hi/docs/setup/production-environment/windows/user-guide-windows-containers.md:101]:
```suggestion
    * Windows नोड पर प्रति पॉड दो कंटेनर, उपयोग करें `docker ps` 
```

File: content/hi/docs/setup/production-environment/windows/user-guide-windows-containers.md [hunk 8]
```diff
@@ -0,0 +1,292 @@
...
+Windows नेटवर्किंग स्टैक की वर्तमान प्लेटफॉर्म सीमाओं के कारण Windows कंटेनर होस्ट उन पर निर्धारित सेवाओं के आईपी तक पहुंचने में सक्षम नहीं हैं।
+केवल Windows पॉड ही सेवा आईपी तक पहुँचने में सक्षम हैं।
+{{< /note >}}
+
+## अवलोकनीयता (Observability)
+
+### कार्यभार से लॉग कैप्चर करना
+
+लॉग्स अवलोकनीयता का एक महत्वपूर्ण तत्व हैं; वे उपयोगकर्ताओं को अंतर्दृष्टि प्राप्त करने में सक्षम बनाते हैं
+कार्यभार के परिचालन पहलू में और समस्याओं के निवारण के लिए एक प्रमुख घटक हैं।
```
Comment (vedant-kakde) [inline content/hi/docs/setup/production-environment/windows/user-guide-windows-containers.md:123]:
```suggestion
वर्कलोड के परिचालन पहलू में और समस्याओं के निवारण के लिए एक प्रमुख घटक हैं।
```

File: content/hi/docs/setup/production-environment/windows/user-guide-windows-containers.md [hunk 9]
```diff
@@ -0,0 +1,292 @@
...
+क्योंकि Windows कंटेनर और Windows कंटेनर के अंदर वर्कलोड Linux कंटेनर से अलग व्यवहार करते हैं,
+उपयोगकर्ताओं को परिचालन दृश्यता को सीमित करते हुए, लॉग एकत्र करने में कठिन समय था।
+उदाहरण के लिए विंडोज वर्कलोड आमतौर पर ETW (Event Tracing for Window) में लॉग इन करने के लिए कॉन्फ़िगर किया गया है।
```
Comment (vedant-kakde) [inline content/hi/docs/setup/production-environment/windows/user-guide-windows-containers.md:126]:
```suggestion
उदाहरण के लिए Windows वर्कलोड आमतौर पर ETW (Event Tracing for Window) में लॉग इन करने के लिए कॉन्फ़िगर किया गया है।
```

File: content/hi/docs/setup/production-environment/windows/user-guide-windows-containers.md [hunk 10]
```diff
@@ -0,0 +1,292 @@
...
+या एप्लिकेशन इवेंट लॉग में प्रविष्टियां पुश करें। 
+[LogMonitor](https://github.com/microsoft/windows-container-tools/tree/master/LogMonitor), Microsoft द्वारा एक खुला स्रोत उपकरण,
+विंडोज कंटेनर के अंदर कॉन्फ़िगर किए गए लॉग स्रोतों की निगरानी के लिए अनुशंसित तरीका है।
```
Comment (vedant-kakde) [inline content/hi/docs/setup/production-environment/windows/user-guide-windows-containers.md:129]:
```suggestion
Windows कंटेनर के अंदर कॉन्फ़िगर किए गए लॉग स्रोतों की निगरानी के लिए अनुशंसित तरीका है।
```

File: content/hi/docs/setup/production-environment/windows/user-guide-windows-containers.md [hunk 11]
```diff
@@ -0,0 +1,292 @@
...
+LogMonitor मॉनिटरिंग इवेंट लॉग, ETW प्रदाताओं और कस्टम एप्लिकेशन लॉग का समर्थन करता है,
+उन्हें `kubectl logs <pod>` द्वारा उपभोग के लिए STDOUT में पाइप करना।
+
+इसके बायनेरिज़ और कॉन्फ़िगरेशन फ़ाइलों को कॉपी करने के लिए LogMonitor गिटहब पेज में दिए गए निर्देशों का पालन करें
+अपने सभी कंटेनरों में और अपने लॉग को STDOUT पर धकेलने के लिए LogMonitor के लिए आवश्यक प्रवेश बिंदु जोड़ें।
+
+## कॉन्फ़िगर करने योग्य कंटेनर उपयोगकर्ता नाम का उपयोग करना
+
+कुबेरनेट्स v1.16 से शुरू होकर, विंडोज कंटेनरों को उनके प्रवेश बिंदुओं और प्रक्रियाओं को चलाने के लिए कॉन्फ़िगर किया जा सकता है
```
Comment (vedant-kakde) [inline content/hi/docs/setup/production-environment/windows/user-guide-windows-containers.md:138]:
```suggestion
कुबेरनेट्स v1.16 से शुरू होकर, Windows कंटेनरों को उनके प्रवेश बिंदुओं और प्रक्रियाओं को चलाने के लिए कॉन्फ़िगर किया जा सकता है
```

File: content/hi/docs/setup/production-environment/windows/user-guide-windows-containers.md [hunk 12]
```diff
@@ -0,0 +1,292 @@
...
+छवि डिफ़ॉल्ट की तुलना में अलग उपयोगकर्ता नाम के साथ।
+इसे हासिल करने का तरीका Linux कंटेनरों के लिए किए जाने वाले तरीके से थोड़ा अलग है।
+इसके बारे में और जानें [यहां](/docs/tasks/configure-pod-container/configure-runasusername/).
```
Comment (vedant-kakde) [inline content/hi/docs/setup/production-environment/windows/user-guide-windows-containers.md:141]:
```suggestion
इसके बारे में और [यहां](/docs/tasks/configure-pod-container/configure-runasusername/) जानें|
```

File: content/hi/docs/setup/production-environment/windows/user-guide-windows-containers.md [hunk 13]
```diff
@@ -0,0 +1,292 @@
...
+
+## समूह प्रबंधित सेवा खातों के साथ कार्यभार पहचान प्रबंधित करना
+
+Kubernetes v1.14 से शुरू होकर, Windows कंटेनर वर्कलोड को ग्रुप मैनेज्ड सर्विस अकाउंट्स (जीएमएसए) का उपयोग करने के लिए कॉन्फ़िगर किया जा सकता है।
+समूह प्रबंधित सेवा खाते एक विशिष्ट प्रकार के सक्रिय निर्देशिका खाते हैं जो स्वचालित पासवर्ड प्रबंधन प्रदान करते हैं,
+सरलीकृत सेवा प्रमुख नाम (एसपीएन) प्रबंधन, और कई सर्वरों पर प्रबंधन को अन्य प्रशासकों को सौंपने की क्षमता।
+जीएमएसए के साथ कॉन्फ़िगर किए गए कंटेनर जीएमएसए के साथ कॉन्फ़िगर की गई पहचान को ले जाने के दौरान बाहरी सक्रिय निर्देशिका डोमेन संसाधनों तक पहुंच सकते हैं।
+Windows कंटेनरों के लिए जीएमएसए को कॉन्फ़िगर करने और उपयोग करने के बारे में और जानें [यहां](/docs/tasks/configure-pod-container/configure-gmsa/).
```
Comment (vedant-kakde) [inline content/hi/docs/setup/production-environment/windows/user-guide-windows-containers.md:149]:
```suggestion
Windows कंटेनरों के लिए जीएमएसए को कॉन्फ़िगर करने और उपयोग करने के बारे में और [यहां](/docs/tasks/configure-pod-container/configure-gmsa/) जानें|
```

File: content/hi/docs/setup/production-environment/windows/user-guide-windows-containers.md [hunk 14]
```diff
@@ -0,0 +1,292 @@
...
+हालाँकि, हम समझते हैं कि कई मामलों में उपयोगकर्ताओं के पास Linux कंटेनरों के लिए पहले से मौजूद बड़ी संख्या में परिनियोजन हैं,
+साथ ही ऑफ-द-शेल्फ कॉन्फ़िगरेशन का एक पारिस्थितिकी तंत्र, जैसे कि सामुदायिक हेल्म चार्ट, और प्रोग्रामेटिक पॉड पीढ़ी के मामले, जैसे कि ऑपरेटरों के साथ।
+उन स्थितियों में, आप नोड चयनकर्ताओं को जोड़ने के लिए कॉन्फ़िगरेशन परिवर्तन करने में संकोच कर सकते हैं।
+विकल्प टेंट्स का उपयोग करना है। क्योंकि क्यूबलेट पंजीकरण के दौरान टेंट सेट कर सकता है,
+केवल Windows पर चलते समय इसे स्वचालित रूप से एक दाग जोड़ने के लिए आसानी से संशोधित किया जा सकता है।
+
+उदाहरण के लिए:  `--register-with-taints='os=windows:NoSchedule'`
+
+सभी Windows नोड्स में एक कलंक जोड़कर, उन पर कुछ भी निर्धारित नहीं किया जाएगा (जिसमें मौजूदा Linux पॉड्स शामिल हैं)।
+विंडोज पॉड को विंडोज नोड पर शेड्यूल करने के लिए,
```
Comment (vedant-kakde) [inline content/hi/docs/setup/production-environment/windows/user-guide-windows-containers.md:180]:
```suggestion
Windows पॉड को Windows नोड पर शेड्यूल करने के लिए,
```

File: content/hi/docs/setup/production-environment/windows/user-guide-windows-containers.md [hunk 15]
```diff
@@ -0,0 +1,292 @@
...
+tolerations:
+    - key: "os"
+      operator: "Equal"
+      value: "windows"
+      effect: "NoSchedule"
+```
+
+### एक ही क्लस्टर में कई Windows संस्करणों को संभालना
+
+प्रत्येक पॉड द्वारा उपयोग किया जाने वाला विंडोज सर्वर संस्करण नोड से मेल खाना चाहिए। यदि आप एकाधिक Windows का उपयोग करना चाहते हैं
```
Comment (vedant-kakde) [inline content/hi/docs/setup/production-environment/windows/user-guide-windows-containers.md:196]:
```suggestion
प्रत्येक पॉड द्वारा उपयोग किया जाने वाला Windows सर्वर संस्करण नोड से मेल खाना चाहिए। यदि आप एकाधिक Windows का उपयोग करना चाहते हैं
```

File: content/hi/docs/setup/production-environment/windows/user-guide-windows-containers.md [hunk 16]
```diff
@@ -0,0 +1,292 @@
...
+एक ही क्लस्टर में सर्वर संस्करण, फिर आपको अतिरिक्त नोड लेबल और नोड चयनकर्ता सेट करना चाहिए।
+
+कुबेरनेट्स 1.17 इसे सरल बनाने के लिए स्वचालित रूप से एक नया लेबल `node.kubernetes.io/windows-build` जोड़ता है।
+यदि आप एक पुराना संस्करण चला रहे हैं, तो इस लेबल को मैन्युअल रूप से Windows नोड्स में जोड़ने की अनुशंसा की जाती है।
+
+यह लेबल विंडोज मेजर, माइनर और बिल्ड नंबर को दर्शाता है जिसे संगतता के लिए मिलान करने की आवश्यकता होती है।
```
Comment (vedant-kakde) [inline content/hi/docs/setup/production-environment/windows/user-guide-windows-containers.md:202]:
```suggestion
यह लेबल Windows मेजर, माइनर और बिल्ड नंबर को दर्शाता है जिसे संगतता के लिए मिलान करने की आवश्यकता होती है।
```

File: content/hi/docs/setup/production-environment/windows/user-guide-windows-containers.md [hunk 17]
```diff
@@ -0,0 +1,292 @@
...
+
+
+### रनटाइम क्लास के साथ सरलीकरण
+
+[RuntimeClass] का उपयोग दाग और सहनशीलता का उपयोग करने की प्रक्रिया को सरल बनाने के लिए किया जा सकता है।
+एक क्लस्टर व्यवस्थापक एक `RuntimeClass` ऑब्जेक्ट बना सकता है जिसका उपयोग इन कलंक और सहनशीलता को समाहित करने के लिए किया जाता है।
+
+
+1. इस फाइल को `runtimeClasses.yml` में सेव करें। इसमें उपयुक्त `nodeSelector` शामिल है
+विंडोज ओएस, आर्किटेक्चर और वर्जन के लिए।
```
Comment (vedant-kakde) [inline content/hi/docs/setup/production-environment/windows/user-guide-windows-containers.md:219]:
```suggestion
Windows OS, आर्किटेक्चर और वर्जन के लिए।
```

**Review by vedant-kakde — CHANGES_REQUESTED** [review]

File: content/hi/docs/setup/production-environment/windows/user-guide-windows-containers.md (hunk 17 above)
Comment (rajatgupta24) [reply content/hi/docs/setup/production-environment/windows/user-guide-windows-containers.md:219]: We are using `संस्करण` for `वर्जन`

File: content/hi/docs/setup/production-environment/windows/user-guide-windows-containers.md [hunk 18]
```diff
@@ -0,0 +1,292 @@
...
+Kubernetes v1.14 से शुरू होकर, Windows कंटेनर वर्कलोड को ग्रुप मैनेज्ड सर्विस अकाउंट्स (जीएमएसए) का उपयोग करने के लिए कॉन्फ़िगर किया जा सकता है।
+समूह प्रबंधित सेवा खाते एक विशिष्ट प्रकार के सक्रिय निर्देशिका खाते हैं जो स्वचालित पासवर्ड प्रबंधन प्रदान करते हैं,
+सरलीकृत सेवा प्रमुख नाम (एसपीएन) प्रबंधन, और कई सर्वरों पर प्रबंधन को अन्य प्रशासकों को सौंपने की क्षमता।
+जीएमएसए के साथ कॉन्फ़िगर किए गए कंटेनर जीएमएसए के साथ कॉन्फ़िगर की गई पहचान को ले जाने के दौरान बाहरी सक्रिय निर्देशिका डोमेन संसाधनों तक पहुंच सकते हैं।
+Windows कंटेनरों के लिए जीएमएसए को कॉन्फ़िगर करने और उपयोग करने के बारे में और जानें [यहां](/docs/tasks/configure-pod-container/configure-gmsa/).
+
+## कलंक और सहनशीलता
+
+उपयोगकर्ताओं को आज के क्रम में कलंक और नोड चयनकर्ताओं के कुछ संयोजन का उपयोग करने की आवश्यकता है
+Linux और विंडोज वर्कलोड को उनके संबंधित ओएस-विशिष्ट नोड्स पर रखें।
```
Comment (rajatgupta24) [inline content/hi/docs/setup/production-environment/windows/user-guide-windows-containers.md:154]:
```suggestion
Linux और Windows वर्कलोड को उनके संबंधित ओएस-विशिष्ट नोड्स पर रखें।
```

File: content/hi/docs/setup/production-environment/windows/user-guide-windows-containers.md [hunk 19]
```diff
@@ -0,0 +1,292 @@
...
+यह संभावना केवल विंडोज उपयोगकर्ताओं पर बोझ डालती है। अनुशंसित दृष्टिकोण नीचे उल्लिखित है,
+इसका एक मुख्य लक्ष्य यह है कि यह दृष्टिकोण मौजूदा Linux वर्कलोड के लिए अनुकूलता को नहीं तोड़ना चाहिए।
```
Comment (rajatgupta24) [inline content/hi/docs/setup/production-environment/windows/user-guide-windows-containers.md:156]:
```suggestion
इसका एक मुख्य लक्ष्य यह है कि यह दृष्टिकोण का मौजूदा Linux वर्कलोड की अनुकूलता को ना तोडे।
```

**Review by rajatgupta24 — CHANGES_REQUESTED** [review]

**Comment by RA489** [comment]
@arsheelsheikh your commits are linked to another user. Please fix that.

**Comment by RA489** [comment]
you may need to change your local Git configuration settings.

**Comment by jihoon-seo** [comment]
/remove-area blog release-eng
/remove-language en es id ja ko pt ru zh
/remove-sig release

**Pull request closed without merging**